<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-07T10:02:45 IST" name="webautomationtestsuite" finished-at="2023-06-07T10:04:50 IST" duration-ms="124645">
    <groups>
    </groups>
    <test started-at="2023-06-07T10:02:45 IST" name="webautomationtestcases" finished-at="2023-06-07T10:04:50 IST" duration-ms="124645">
      <class name="testcases.create_w3schoolprofile">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.create_w3schoolprofile@5b03b9fe]" started-at="2023-06-07T10:02:47 IST" name="setup" finished-at="2023-06-07T10:03:11 IST" duration-ms="24285" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="createprofile(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.create_w3schoolprofile@5b03b9fe]" started-at="2023-06-07T10:03:11 IST" name="createprofile" data-provider="bvttestdata" finished-at="2023-06-07T10:03:36 IST" duration-ms="24691" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[vinayquiet@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bunny@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[vinay]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Create Profile Testcase is Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Create Profile Testcase is Failed
at org.testng.Assert.fail(Assert.java:87)
at testcases.create_w3schoolprofile.createprofile(create_w3schoolprofile.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: //input[@placeholder='@"nickname"']
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [087aeeda-76ee-4db6-8aaf-8dd2ebaf3fe3, findElement {using=xpath, value=//input[@placeholder='@"nickname"']}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 113.0.2, moz:accessibilityChecks: false, moz:buildID: 20230522134052, moz:debuggerAddress: 127.0.0.1:17790, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 13824, moz:profile: C:\Users\BSA\AppData\Local\..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://127.0.0.1:17790/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 087aeeda-76ee-4db6-8aaf-8dd2ebaf3fe3
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at utilities.CommonFunctions.cf_webelementsendkeys(CommonFunctions.java:63)
at testcases.create_w3schoolprofile.createprofile(create_w3schoolprofile.java:23)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createprofile -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.create_w3schoolprofile@5b03b9fe]" started-at="2023-06-07T10:03:37 IST" name="teardown" finished-at="2023-06-07T10:03:41 IST" duration-ms="4494" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.create_w3schoolprofile -->
      <class name="testcases.update_w3schoolprofile">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.update_w3schoolprofile@771a660]" started-at="2023-06-07T10:03:41 IST" name="setup" finished-at="2023-06-07T10:03:56 IST" duration-ms="15072" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="updateprofile(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.update_w3schoolprofile@771a660]" started-at="2023-06-07T10:03:56 IST" name="updateprofile" data-provider="bvttestdata" finished-at="2023-06-07T10:04:20 IST" duration-ms="24068" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[vinayquiet@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bunny@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[vinay]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateprofile -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.update_w3schoolprofile@771a660]" started-at="2023-06-07T10:04:20 IST" name="teardown" finished-at="2023-06-07T10:04:24 IST" duration-ms="3228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.update_w3schoolprofile -->
      <class name="testcases.delete_w3schoolprofile">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.delete_w3schoolprofile@242b836]" started-at="2023-06-07T10:04:24 IST" name="setup" finished-at="2023-06-07T10:04:34 IST" duration-ms="10767" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="deleteprofile(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.delete_w3schoolprofile@242b836]" started-at="2023-06-07T10:04:34 IST" name="deleteprofile" data-provider="bvttestdata" finished-at="2023-06-07T10:04:47 IST" duration-ms="12888" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[vinayquiet@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bunny@123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[vinay]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Delete Profile Testcase is Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Delete Profile Testcase is Failed
at org.testng.Assert.fail(Assert.java:87)
at testcases.delete_w3schoolprofile.deleteprofile(delete_w3schoolprofile.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: User Logged in Failed. expected [My learning | W3Schools] but found [Log in - W3Schools]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testcases.delete_w3schoolprofile.deleteprofile(delete_w3schoolprofile.java:23)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteprofile -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.delete_w3schoolprofile@242b836]" started-at="2023-06-07T10:04:47 IST" name="teardown" finished-at="2023-06-07T10:04:50 IST" duration-ms="2274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.delete_w3schoolprofile -->
    </test> <!-- webautomationtestcases -->
  </suite> <!-- webautomationtestsuite -->
</testng-results>
